packages/xcode-build-server/package.yaml

```diff
@@ -0,0 +1,18 @@
+---
+name: xcode-build-server
+description: A server to bridge Xcode build system outside of Xcode
+homepage: https://github.com/SolaWing/xcode-build-server
+licenses:
+  - MIT
+languages:
+  - Swift
+categories:
+  - LSP
+
+source:
+  id: pkg:github/SolaWing/xcode-build-server@v1.1.0
+  build:
+    run: "echo already built"
```

Comment:
that's unnecessary, `source.build` is actually not required. 

If the release already contains binaries that are read to use, you have to specify them as `assets`. [See this package for an example.](https://github.com/mason-org/mason-registry/blob/main/packages/actionlint/package.yaml)